src/store/dao/actions.js

```diff
@@ -191,6 +224,31 @@ export const updateDAOSettings = async function (context, { docId, data, alerts,
             ]]
           }
         }]
+      : []),
+
+    ...(!daoSettings.upvoteElectionId && data.communityVotingMethod === 'UPVOTE'
+      ? [
+          {
+            account: this.$config.contracts.dao,
+            name: 'createupvelc',
+            data: {
+              dao_id: docId,
+              ...upvoteData
+            }
+          }
+        ]
+      : []),
+    ...(daoSettings.upvoteElectionId && data.communityVotingMethod === 'UPVOTE'
```

Comment:
Please use constant for `UPVOTE` instead of a string. That constant should be used wherever you need this `UPVOTE` value

Reply:
If it is used just here, then it can be just in this file. However, in most cases it is the best to have constants shared. I think for this instance, you can update this file, create global constants and make a new task to refactor.